eng/scripts/aggregate_javadoc_configuration.txt

```diff
@@ -0,0 +1,94 @@
+# File containing frequently updated aggregate JavaDoc configuration
+
+# Adds the following groups to the aggregate JavaDoc
+# Group;Title;Package
```

Comment:
Future enhancement, provide a utility that can update these automatically. Should be able to do something where the module-info of a library is inspected for exports, not exports to, and create the colon delimited list of packages included in Javadocs.

Reply:
Created https://github.com/Azure/azure-sdk-for-java/issues/30131